KarhooUISDK/CommonUI/BottomSheet/KarhooBottomSheet.swift

```diff
@@ -23,104 +23,93 @@ struct KarhooBottomSheet<Content: View>: View {
    
     // MARK: - Body
     var body: some View {
-        VStack(spacing: 0.0, content: {
+        ZStack {
```

Comment:
What are your thoughts on extracting the sub-views to specified getters/methods and using `body` method as higher-level function? This way we could achieve some level of readability.